tests/ci/runpytest.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
```

Comment:
What is the function of this file? and how is this related to submitpytest?

Reply:
This is the script that submitpytest submits to the AzureML service.  It's set up so that when pytest corrupts the AML service environment variables that the log files can still be returned to devops for publishing.

Reply:
I see. 

A general question in relation to this, how do you want to show all this pipelines? I think this is really nice. One option is to show this on the test markdown. Besides, this could be a nice history to showcase AzureML. Maybe we could do a blog post. What do you think?

Reply:
Agree. I also set up the experiment name on the azure portal so that the pipeline name shows up on the azure portal as an experiment. With the initial submissions, there was no way to correllate the devops info with the azure portal and that was important for usage and for debugging.  I would like to make the info on the portal more useful at some point.